tests/functional/tests/backup.py

```diff
@@ -17,21 +16,27 @@
 class BackupTest(unittest.TestCase):
     """Code for backup test."""
 
+    def setUp(self) -> None:
+        zaza.get_or_create_libjuju_thread()
+        model_name = zaza.model.get_juju_model()
+        self.model = COUModel(model_name)
```

Comment:
Don't we need to connect to the model by running `self.model.connect()`?

Reply:
I do not think there is any need for that since it's calling the same functions as this. But I agree that it could be confusing, that we are not connecting to model.